package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "vscode-hapijs-snippets",
-  "publisher": "deerawan",
-  "version": "2.0.0",
+  "publisher": "deerawan, siddharth",
```

Comment:
I'm afraid we can't put extra name here since this is an identifier for vscode marketplace.